server/src/utils/store.js

```diff
@@ -0,0 +1,75 @@
+import Redis from 'redis';
```

Comment:
This is fine for now- eventually we'd probably want to break this out into their individual files and have `store.js` just be the exporter for `Redis` and `Memory`. Even better we'd move the entire store class outside of utils into its own root folder.

Reply:
Their is now a file for each in the store folder.